local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,42 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(options) {
+    var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+    console.log('Launching Dev Tools...');
+    opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+        if (err) {
+            console.error('Google Chrome exited with error:', err);
+        }
+    });
+}
+
+function escapePath(path) {
+  return path.indexOf(' ') > -1
+    ? '"' + path + '"' // " Can escape paths with spaces in OS X, Windows, and *nix
+    : path;
+}
+
+function launchDevTools(options, isChromeConnected) {
+    // Explicit config always wins
+    var customDebugger = process.env.REACT_DEBUGGER;
+    if (customDebugger) {
+        var projects = options.projectRoots.map(escapePath).join(' ');
+        var command = customDebugger + ' ' + projects;
+        console.log('Starting custom debugger by executing: ' + command);
+        child_process.exec(command, function (error, stdout, stderr) {
```

Comment:
Should we use [`child_process.spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) here instead? If I'm not mistaken, [`child_process.exec`](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) will buffer the output of `command`, which might consume a lot of memory if `command` runs for a while.